packages/cozy-client/src/models/file.spec.js

```diff
@@ -215,6 +215,11 @@ describe('File Model', () => {
       expect(() => fileModel.splitFilename({})).toThrow()
       expect(() => fileModel.splitFilename({ name: null })).toThrow()
       expect(() => fileModel.splitFilename({ name: '' })).not.toThrow()
+      try {
+        fileModel.splitFilename({})
+      } catch (error) {
+        expect(error.message).toEqual('file should have a name property')
```

Comment:
Care, here if `splitFileName` does not throw, then the test will still pass because the expect won't be called. Prefere to use `.toThrow('message')` instead.